office/watson/Portfile

```diff
@@ -24,15 +24,16 @@ categories          office python
 maintainers         {gmail.com:herby.gillot @herbygillot} \
                     openmaintainer
 
-checksums           rmd160  cae4d7d3d9f4e1a839fc49d318b9c5183597b3e7 \
-                    sha256  a665775a76fb3ac464153e10991577a38ca938adae2142a24c9d1b1234db95f0 \
-                    size    8895378
+checksums           rmd160  e1c5e0c4f97c519e00e62e1737e5151d29887baf \
+                    sha256  204384dc04653e0dbe8f833243bb833beda3d79b387fe173bfd33faecdd087c8 \
+                    size    8895400
+
+python.default_version  39
```

Comment:
The current default Python version in MacPorts today is `310` (Python 3.10). Is there a reason why we're setting it to 3.9?

Reply:
Right. I actually tried updating to Python 3.10 but right now there is no py310-click-didyoumean in macports which would be a dependency. We can probably look at introducing that subport as part of this upgrade but I was mainly interested in a quick fix for the currently broken watson port.

Reply:
I think it's fine to add a separate, preceding commit to this PR that just adds the `310` subport to `py-click-didyoumean`, if just doing that will work.  But if additional changes are also required for `didyoumean`'s dependencies, then sure, I suppose we can just continue with using 3.9 for `watson`.

In general, we usually want to move Python projects to whatever is the current default version as we update them over time.

Reply:
Sure, I'll try this out tomorrow. We can also update click-didyoumean to [v0.3.0](https://github.com/click-contrib/click-didyoumean/releases/tag/v0.3.0) then while we're at it.

Reply:
Sounds good, thanks.

Reply:
I added a separate commit that updates py-click-didyoumean and introduces py310-click-didyoumean. It seems to work well, I briefly tested the didyoumean functionality of watson:
```
 ~ % watson statu
[...]
Did you mean one of these?
    status
    start
    restart
```